packages/uhk-web/src/app/components/device/mouse-speed/mouse-speed.component.ts

```diff
@@ -8,5 +17,137 @@ import { Component } from '@angular/core';
         'class': 'container-fluid'
     }
 })
-export class MouseSpeedComponent {
+export class MouseSpeedComponent implements OnInit, OnDestroy {
+    public moveProps = [
+        {
+            prop: 'mouseMoveInitialSpeed',
```

Comment:
The agent currently template driven and I feel if you are create a new component and set property bindings than this part of code should be in the template for the consistency.

Reply:
I'd rather keep them in here. I understand it creates more mess in the component itself but the other alternative is just a ton of repeated code in the template. I think this is a more maintainable solution.